hazyCalculator.js

```diff
@@ -1,18 +1,31 @@
 function isSkippedValue(value) {
+  if (value === undefined || value === '') {
+    return true
+  }
+  const operators = ['+', '-', '*', '/']
+  if (operators.includes(value)) {
+    return false
+  }
   return !value
 }
 
 function isNumericValue(value) {
+  if (value === null) {
+    return true
+  }
+  else if (value == '' || value === undefined) {
+    return false
+  }
   return !isNaN(value)
 }
 
 function isNothingValue(value) {
-  return value === null
+  return value === null || value === undefined
 }
 
 function isAcceptableValue(value) {
   const operators = ['+', '-', '*', '/']
-  return typeof value === Number || operators.includes(value)
+  return typeof Number(value) === 'number' || operators.includes(value)
```

Comment:
We have a function called `isNumericValue` wouldn't we want to use this here instead of having a separate definition of when a value is numeric?

We also aren't accounting for skipped values here. They are technically acceptable since we know they will be skipped.